selenium-tests/src/test/java/za/co/absa/BaseTest.java

```diff
@@ -0,0 +1,192 @@
+package za.co.absa;
+
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.concurrent.TimeUnit;
+
+import org.openqa.selenium.*;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.ExpectedCondition;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.BeforeMethod;
+
+public class BaseTest {
+  WebDriver driver;
+  private DateTimeFormatter dtf = DateTimeFormatter.ofPattern("yyyy_MM_dd-HH_mm_ss");
+  String postfix = dtf.format(LocalDateTime.now());
+
+  By schemasView = By.id("__navigation0---schemaMainView");
+  By datasetsView = By.id("__navigation0---datasetMainView");
+  By mappingTablesView = By.id("__navigation0---mappingTableMainView");
+  private By nameField = By.name("username");
+  private By passwordField = By.name("password");
+  String username = "user";
+  private String password = "changeme";
```

Comment:
1. Generally not a good style to have fields with different visibility mixed. First public, then protected, then default, finally private.
2. These seem like constants that duplicate the ones in `Base`. I'd extract them in a Constants class and make them all `public static final`.